tests/test_prettier.py

```diff
@@ -424,3 +431,27 @@ def test_asyncpg_record():
 
 def test_dict_type():
     assert pformat(type({1: 2})) == "<class 'dict'>"
+
+
+@pytest.mark.skipif(SQLAlchemyBase is None, reason='sqlalchemy not installed')
```

Comment:
This needs to be an extra requirement like numpy, right now it looks like this test is always skipped in CI.

Reply:
https://github.com/samuelcolvin/python-devtools/pull/94/commits/edc2ae109084090dfc634fca7a1b585833093b52#diff-27c5febcc9ce93e8e5341f263b7d68962733b8a2a86c583723d025a1db39d7a3R12

Sorry, I am newbie, I don't know meaning of extra requirement. I did add sqlalchemy in above requirements.txt file. I hope its true.

Reply:
Yes what you added looks perfect